test/lib/mutation_source_test.cc

```diff
@@ -1386,20 +1386,23 @@ void test_slicing_with_overlapping_range_tombstones(populate_fn_ex populate) {
     }
 }
 
-void run_mutation_reader_tests(populate_fn_ex populate) {
-    test_slicing_and_fast_forwarding(populate);
+void run_mutation_reader_tests(populate_fn_ex populate, bool run_fast_forwarding_tests) {
     test_date_tiered_clustering_slicing(populate);
-    test_fast_forwarding_across_partitions_to_empty_range(populate);
-    test_clustering_slices(populate);
     test_mutation_reader_fragments_have_monotonic_positions(populate);
-    test_streamed_mutation_forwarding_across_range_tombstones(populate);
-    test_streamed_mutation_forwarding_guarantees(populate);
     test_all_data_is_read_back(populate);
     test_streamed_mutation_slicing_returns_only_relevant_tombstones(populate);
-    test_streamed_mutation_forwarding_is_consistent_with_slicing(populate);
     test_range_queries(populate);
     test_query_only_static_row(populate);
     test_query_no_clustering_ranges_no_static_columns(populate);
+
+    if (run_fast_forwarding_tests) {
```

Comment:
Botond had a suggestion to implement fast forwarding using the `make_forwardable()` wrapper. Then you won't have to disable these tests.

Reply:
```
        if (fwd == streamed_mutation::forwarding::yes) {
            return make_forwardable(std::move(mr));
        } else {
            return mr;
        }

```

Reply:
It seems that the forwarding tests pass even without wrapping using `make_forwardable()`.
Could we just make a method that asserts forwarding and overwrite it in tests? That way it will break normally as it is right now (forwarding not supported), but continue past that assertion in the mutation source tests

Reply:
```
if (disallow_forwarding(fwd)) {
    throw std::runtime_error("streamed_mutation::forwarding::yes unsupported in virtual tables");
}
```

Reply:
If it works, then let's allow it.

Reply:
so just remove the if statement completely?